developer_manual/app/publishing.rst

```diff
@@ -69,3 +69,33 @@ Respect the users
 * App authors must provide means to contact them, be it through a bug tracker, forum or mail.
 
 Apps which break the guidelines will lose their 'approved' or 'official' state; and might be blocked from the app store altogether. This also has repercussions for the author, especially in case of security concerns, he/she might find themselves blocked from submitting applications.
+
+Putting your repo in the Nextcloud Project
+------------------------------------------
+We're always delighted to hear app develpers are interested in moving their app into the Nextcloud Project in `github.com/nextcloud <https://github.com/nextcloud>`_! There are benefits for users and developers in being there. However, it comes with some requirements as well.
+
+Benefits
+^^^^^^^^
+
+* You can use the tools and bots we have set up, including translations and such
+* Everybody in the Nextcloud project can more easily contribute
+* Your visibility to app developers increases
+* Users can expect apps in our project to be better maintained
+
+Requirements
+^^^^^^^^^^^^
+To deliver on the promises above, we have two simple rules.
+
+* You participate in our `collaborative development model <../general/code-of-conduct.html>`_
+* When you are no longer active, our admins can decide to hand over maintainership to another contributor
+
+We want to make sure that when you find other things in life which are more urgent or otherwise are unable to help your project anymore, it does not become 'dead code' as long as there are people who want to keep it alive. This is not fair to users, who would be forced to remove the app and install another.
+
+Please note that the role of a maintainer is not to be the most active or prolific contributor to a project! Worlds' `most famous maintainer of an open source project <https://en.wikipedia.org/wiki/Linus_Torvalds>`_ has not appeared in the top-100 of contributors by lines of code for decades, yet he runs the largest single software development project in the history of mankind. Being friendly, welcoming and responsive are what it takes to be a successful maintainer. Not being the most brilliant developer ever, or spending nights and weekends coding!
```

Comment:
I’d prefer we don’t put Linus Torvalds even more on a pedestal. :) We don’t really need a specific example there.

Reply:
```suggestion
Please note that the role of a maintainer is not to be the most active or prolific contributor to a project! Being friendly, welcoming and responsive are what it takes to be a successful maintainer. Not being the most brilliant developer ever, or spending nights and weekends coding!
```